src/middlewared/middlewared/plugins/docker/update.py

```diff
@@ -32,6 +38,13 @@ class Config:
         Str('dataset', required=True),
         Str('pool', required=True, null=True),
         Bool('nvidia', required=True),
+        List('address_pools', items=[
+             Dict(
+                 'address_pool',
+                 IPAddr('base', cidr=True),
+                 Int('size', validators=[Range(min_=1, max_=32)])
```

Comment:
Sorry but what is `size` here?

Reply:
`size` is essentially the number of IP addresses the pool can contain starting from base address. So if it s 24, then each network created can accommodate 256 interfaces

Reply:
That's incredibly confusing :smile:. So much so that I had to go look it up (shocker, the documentation isn't very forthcoming). The `size` parameter sub-divides the base prefix (using our defaults `/16`) into `/27`'s. A /27 is 30 usable IPs, so each time more than 30 ips are used within a subnet, it'll start to use the next subnet from the base pool.